scripts/docker-env

```diff
@@ -34,7 +34,7 @@ def build(args):
         subprocess.run(
             build_docker_command(
                 args.docker_file,
-                ["make", "x86_64.rpm"],
+                ["make", "clean", "x86_64.rpm"],
```

Comment:
These flags need to be conditional based on a flag. We can't rebuild from scratch every time...

Reply:
we still need these changes

Reply:
Dude, we still need these changes....

Reply:
I know, I have not forgotten! That is why i did not resolve this conversation!